src/similar-npm-packages.ts

```diff
@@ -501,6 +521,16 @@ export const categories: Categories = {
     ],
   },
 
+  "react-slider": {
+    name: "Slider components for React",
+    tags: [
+      { tag: "react", weight: Weight.NORMAL },
+      { tag: "slider", weight: Weight.MAX },
+      { tag: "range", weight: Weight.NORMAL },
+    ],
+    similar: ["rc-slider", "react-slider", "react-input-slider", "rheostat"],
```

Comment:
https://www.npmjs.com/package/react-input-range
https://www.npmjs.com/package/react-rangeslider
https://www.npmjs.com/package/react-range

Seem to be all more popular than the last two. 
Let's pick one's with better quality of docs, community, tests (keeping a max of 4 or 5).

Reply:
`react-input-range` and `react-range` are 👍
`react-rangeslider` has a lot of open issues and no updates for 3 years 👎

Reply:
`react-input-range` is also "dead": 80 issues, no updates for 3 years.

Reply:
This is where I ended up:
`["rc-slider", "react-slider", "react-range", "react-input-slider"]`